### PR TITLE
Create Practice.cs

### PR DESCRIPTION
**Pull request by charientism**

### CODE REVIEW

**Comment by Irdakuh** [comment]
This is the best code I've ever seen

File: Practice.cs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+			while (original != 0)
+			{
+				int digit = original % 10;
+				reverse = reverse*10 + digit;
+				original = original / 10;
+			}
+			return reverse;
+		}
+
+		private static bool IsPalindrome(int original, int reverse)
```
Comment (charientism) [inline Practice.cs:20]: Practice comment.

File: Practice.cs [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		{
+			return original == reverse;
+		}
+
+
+		public static bool IsOrCanBePalindrome(int number)
+		{
+			const int MaxPalindrome = 1000000;
```
Comment (charientism) [inline Practice.cs:28]: Practice comment 2

**Review by charientism — COMMENTED** [review]
Looks great!

**Pull request merged**